assets/src/edit-story/components/panels/test/textStyle.js

```diff
@@ -43,10 +43,19 @@ function Wrapper({ children }) {
   return (
     <FontContext.Provider
       value={{
-        state: { fonts: [{ name: 'ABeeZee', value: 'ABeeZee' }] },
+        state: {
+          fonts: [
+            { name: 'ABeeZee', value: 'ABeeZee' },
+            {
+              name: 'Neu Font',
+              value: 'Neu Font',
+              service: 'foo.bar.baz',
+              fallbacks: ['fallback1'],
+            },
+          ],
+        },
         actions: {
-          getFontWeight: () => [{ name: 'Normal1', value: '400' }],
-          getFontFallback: () => 'fallback1',
+          getFontWeights: () => [{ name: 'Normal1', value: '400' }],
```

Comment:
Out of curiousity: why not just have `weights: [...]` in the `fonts` structure above?

Reply:
This action is currently used for the font weights dropdown, to have a human readable list of font weights (e.g. 700 -> "bold")

Reply:
Ok, it's fine to keep. But imho this would work just as well as a static util function. Up to you.

Reply:
Simplified in 12ca64f11158818eda096ff701de52bb7e74652f.